utils/mock-data-loader/gef/eligibilityCriteria.js

```diff
@@ -73,6 +73,77 @@ const ELIGIBILITY_CRITERIA = [
       },
     ],
   },
+  {
+    version: 4,
```

Comment:
Same as previous comment - should this be version 2 instead of 4? existing EC is version 1.5